functions/src/payment.js

```diff
@@ -1,3 +1,5 @@
+// TODO: Rename this file to payment.ts (or payment/index.ts) + refactor to make proper use of TypeScript
```

Comment:
[TODO] Rename this file to `payment.ts` (or `payment/index.ts`) + refactor to make proper use of TypeScript